### PR TITLE
Fix group chat issues

### PR DESCRIPTION
**Pull request by jrainville**
This was a special ops task Iuri gave to me to test and fix any issue I found in the group chat.

- I found is that if someone leaves the group, they weren't removed from the user list. Now they are.
- The add new member to group didn't work
- Fix context menu
- Fix pinned messages popup from the group info popup

### CODE REVIEW

**Comment by jrainville** [comment]
I made this a draft, because I realised there were other issues not listed in the test list. I'll add more commits and let you guys know when you can review

**Comment by jrainville** [comment]
alright, you guys can review and test

**Comment by abhi-status** [comment]
Thanks Jo!

Could you please look into the group chat name change not being reflected in the main Chat view ?

image

**Comment by jrainville** [comment]
@abhi-status I fixed it. I also found that changing the channel name didn't update it in another instance, so I fixed that too.

You can test again.

File: ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml [hunk 1]
```diff
@@ -170,40 +171,35 @@ StatusModal {
...
+            property int pinnedCount: popup.chatContentModule.pinnedMessagesModel.count
 
             id: pinnedMessagesBtn
             visible: pinnedCount > 0
             //% "Pinned messages"
             text: qsTrId("pinned-messages")
             currentValue: pinnedCount
-            onClicked: Global.openPopup(pinnedMessagesPopupComponent, {store: popup.store})
+            onClicked: {
+                popup.store.messageStore.messageModule = popup.chatContentModule.messagesModule
```
Comment (stefandunca) [inline ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml:182]: I see this as a requirement of the PinnedMessagePopup?  Is the update of module instance coupled with user actions?

**Review by stefandunca — APPROVED** [review]

**Review by saledjenic — APPROVED** [review]

**Pull request merged**